### PR TITLE
Comp/College link to mentors

### LINKED ISSUE #187: Modify Colleges - hook up to Mentors page

### PR DESCRIPTION
**Pull request by jessewoo**
## Proposed changes
Colleges to mentors

Close #187

## Types of changes

- [x ] I've labeled this PR with appropriate labels (`comp/*` component, bug/feature, documentation and other less important ones).

## Checklist
_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [ x] I've linked the proper Github Issue(s) to my PR
- [ x] Lint and unit tests pass locally with my changes
- [x ] I have added tests (unit, integration...etc.) that prove my fix is effective or that my change works
- [x ] I have added or updated necessary documentation (if appropriate). You can search, for instance using the [`silver searcher`](https://github.com/ggreer/the_silver_searcher) `ag <STRING>` to find all usages in the repository.
- [ x] I have notified the team of any breaking or important changes

### CODE REVIEW

File: frontend/src/mock-data/colleges.js [hunk 1]
```diff
@@ -1,167 +1,198 @@
...
   },
   {
     name: 'Cal Tech',
     abbr: 'caltech',
+    active: false,
     imageUrl: 'http://jessewoo.github.io/images/ismp/colleges/caltech.jpg'
   },
   {
-    name: 'Carnegie Melon',
+    name: 'Carnegie Melon University',
```
[the remark this answers is not in the document]
Comment (jessewoo) [reply frontend/src/mock-data/colleges.js]: fixed.

**Comment by jessewoo** [comment]
@douglaschan32167 - we can kill off the school description.

**Comment by jessewoo** [comment]
@douglaschan32167 - description of colleges killed off.

File: frontend/src/pages/BlogSearchResults/BlogListPage.jsx [hunk 2]
```diff
@@ -106,7 +106,7 @@ const BlogSearch = ({ term }) => {
             thumbnail_url: p && p.thumbnail_url,
             title: blog.title_content,
             blurb: blog.body_content,
-            blog_url: `/blogpost/${blog.id}`,
```
Comment (douglaschan32167) [inline frontend/src/pages/BlogSearchResults/BlogListPage.jsx:109]: Going to go ahead and change this back before merging.

File: frontend/src/pages/BlogSearchResults/BlogListPage.jsx [hunk 3]
```diff
@@ -128,7 +128,7 @@ const BlogSearch = ({ term }) => {
       },
       error => {
         const errorObj = error.response;
-        setErrorMsg(`${errorObj ? errorObj.error : error}`);
```
Comment (douglaschan32167) [inline frontend/src/pages/BlogSearchResults/BlogListPage.jsx:131]: also will change this back

**Comment by douglaschan32167** [comment]
@jessewoo going to merge this so that we can get it to the interns who will be testing our website tonight.

**Comment by douglaschan32167** [comment]
@kailanjian @misunchoi @vickyadebona @albertjke I merged this because I wanted to get the fix for the mentors page in before we push to staging and give the site to the interns, but feel free to comment on this still and we can make changes based on what you find.

File: frontend/src/pages/BlogSearchResults/BlogListPage.jsx (hunk 3 above)
Comment (misunchoi) [reply frontend/src/pages/BlogSearchResults/BlogListPage.jsx:131]: I added this fix to prevent it from breaking, not sure if there's a better solution

**Pull request merged**